src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/GnosisControlSafeDialog.tsx

```diff
@@ -137,6 +140,23 @@ const validationSchema = yup.object().shape({
         then: yup.string().required(() => MSG.requiredFieldError),
         otherwise: false,
       }),
+      nft: yup.object().shape({
```

Comment:
I think this validation may need to be `.nullable()` to fix not being able to add more transactions when you are not transferring NFTs

Reply:
Good advice!!